From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Wed, 12 Oct 2016 06:08:46 -0400
Subject: install cmake files without executable bit set

cmake is exciting but it's not *that* exciting, and besides lintian
complains:

W: libgpgme-dev: executable-not-elf-or-script
     usr/lib/x86_64-linux-gnu/cmake/Gpgmepp/QGpgmeConfig.cmake

none of the other cmake files on a debian system appear to be intalled
with executable provisiona either.
---
 lang/cpp/src/Makefile.am | 4 ++--
 lang/qt/src/Makefile.am  | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/lang/cpp/src/Makefile.am b/lang/cpp/src/Makefile.am
index e65a875..0165074 100644
--- a/lang/cpp/src/Makefile.am
+++ b/lang/cpp/src/Makefile.am
@@ -82,9 +82,9 @@ GpgmeppConfig.cmake: GpgmeppConfig.cmake.in
 
 install-cmake-files: GpgmeppConfig.cmake GpgmeppConfigVersion.cmake
 	-$(INSTALL) -d $(DESTDIR)$(libdir)/cmake/Gpgmepp
-	$(INSTALL) GpgmeppConfig.cmake \
+	$(INSTALL) -m 644 GpgmeppConfig.cmake \
         $(DESTDIR)$(libdir)/cmake/Gpgmepp/GpgmeppConfig.cmake
-	$(INSTALL) GpgmeppConfigVersion.cmake \
+	$(INSTALL) -m 644 GpgmeppConfigVersion.cmake \
         $(DESTDIR)$(libdir)/cmake/Gpgmepp/GpgmeppConfigVersion.cmake
 
 uninstall-cmake-files:
diff --git a/lang/qt/src/Makefile.am b/lang/qt/src/Makefile.am
index c15da18..1ae596a 100644
--- a/lang/qt/src/Makefile.am
+++ b/lang/qt/src/Makefile.am
@@ -232,9 +232,9 @@ $(camelcase_headers): Makefile.am
 
 install-cmake-files: QGpgmeConfig.cmake QGpgmeConfigVersion.cmake
 	-$(INSTALL) -d $(DESTDIR)$(libdir)/cmake/Gpgmepp
-	$(INSTALL) QGpgmeConfig.cmake \
+	$(INSTALL) -m 644 QGpgmeConfig.cmake \
         $(DESTDIR)$(libdir)/cmake/Gpgmepp/QGpgmeConfig.cmake
-	$(INSTALL) QGpgmeConfigVersion.cmake \
+	$(INSTALL) -m 644 QGpgmeConfigVersion.cmake \
         $(DESTDIR)$(libdir)/cmake/Gpgmepp/QGpgmeConfigVersion.cmake
 
 uninstall-cmake-files:
